packages/checkout/sdk/src/types/network.ts

```diff
@@ -3,6 +3,16 @@ import { ChainId } from './chains';
 import { NetworkInfo } from './networkInfo';
 import { TokenInfo } from './tokenInfo';
 
+/**
+ * Interface representing the parameters for {@link Checkout.addNetwork}.
+ * @property {Web3Provider} provider - The provider to connect to the network.
+ * @property {ChainId} chainId - The ID of the network to add.
+ */
+export interface AddNetworkParams {
+  provider: Web3Provider;
+  chainId: ChainId;
+}
```

Comment:
When we add docs for this method it would be good to note that we only support adding Immutable zkEVM and/or Immutable zkEVM Testnet for now